howto/install-appliance/install-on-aws.md

```diff
@@ -140,12 +140,31 @@ When the instance is successfully launched, you can find its public IP address i
 
 Connect to the virtual machine hosting the appliance using SSH. To do so, use the user name `ubuntu` and provide the path to your private key file. See [Connect to your Linux instance using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html) for instructions on how to connect.
 
-## Finish the installation
+## Install additional packages
 
-Perform the following steps to finish the appliance installation on the virtual machine. If you are not already familiar with how to perform these steps, see {ref}`tut-installing-appliance` for detailed instructions.
+Some preparation is required for using Anbox Cloud that involves installing additional packages. To prepare your machine for further steps, run:
 
-1. Prepare the virtual machine and install additional required packages. See {ref}`sec-prepare-machine` for instructions.
-1. Initialise the appliance. See {ref}`sec-initialise-appliance` for instructions.
-1. Register your Ubuntu SSO account with the appliance dashboard. See {ref}`sec-register-dashboard` for instructions.
+    anbox-cloud-appliance prepare-node-script
+    anbox-cloud-appliance prepare-node-script | sudo bash -ex
```

Comment:
If I understand correctly, these two instructions are "either-or". We may need to add a few lines to explain which one to use.

Reply:
You are right, the first one is not needed unless you want to look at the script. That can be for the deeper learning experience detailed in the linked tutorial. For a how-to guide that intends to run through the exact steps of deployment, applying the script is enough.